stellarphot/photometry/photometry.py

```diff
@@ -187,8 +196,31 @@ def single_image_photometry(ccd_image, sourcelist, camera, observatory_location,
         raise ValueError(f"input_coordinates ({use_coordinates}) must be either "
                          "'pixel' or 'sky'.")
 
-    # Check CCDData headers before proceeding
+    # Set up logging (retrieve a logger but purge any existing handlers)
+    root_logger = logging.getLogger()
+    for handler in root_logger.root.handlers[:]:
+        root_logger.root.removeHandler(handler)
+
+    logger = logging.getLogger("single_image_photometry")
+    logger.setLevel(logging.INFO)
+    for handler in logger.handlers[:]:
```

Comment:
You have more recent logging experience than me -- is this necessary to remove some default handlers or something?

Reply:
I think this may be code left over from when I was trying to understand why there were extra logging statements coming to the console.  I removed these lines, I don't think they are necessary... the logger, created the lines before, should not have handlers set up yet.